runtimes/BUILD.bazel

```diff
@@ -25,6 +28,39 @@ RUNTIMES = {
     for runtime in RUNTIMES.keys()
 ]
 
+write_file(
+    name = "zml.txt",
+    out = "zml.txt",
+    content = ["ZML loves you <3"],
+)
+
+tar(
```

Comment:
and this ?

Reply:
needed because else `oci_image` complains that `layers` is empty